docs/json_rpc_api.mediawiki

```diff
@@ -362,6 +362,10 @@ the method name for further details such as parameter and return information.
 |Y
 |Asks the daemon to rebroadcast the winners of the voting lottery.
 |-
+|[[#regentemplate|regentemplate]]
+|Y
```

Comment:
If it's available to limited users (which I think it should be), it needs to be added to the `rpcLimited` map in `rpcserver.go`.  Commands are not available to limited users by default.